packages/perennial/contracts/multiinvoker/MultiInvokerRollup.sol

```diff
@@ -41,6 +42,10 @@ contract MultiInvokerRollup is IMultiInvokerRollup, MultiInvoker {
     /// @dev Index lookup of above array for constructing calldata
     mapping(address => uint256) public addressLookup;
 
+    /// @dev magic byte to prepend to calldata for the fallback. 
+    /// Prevents public fns from being called by arbitrary fallback data
+    bytes8 public constant INVOKE_ID = hex'45';
```

Comment:
Can we add a test which fetches this value?

Reply:
changed it to uint8 ... = 73 also because of how its required